ui/app/manager/src/pages/page-alarms.ts

```diff
@@ -0,0 +1,1133 @@
+                                            span.parentElement.style.width = '1%';
+                                            span.parentElement.style.padding = '2px';
+                                        }
+                                        break;
+                                    case 'Open':
+                                        if(columnIndex == 2  && span.parentElement?.style){
+                                            span.innerHTML = '<span style="color: white;' +
+                                                'padding: 4px;' +
+                                                'background-color: mediumblue;' +
+                                                'border-radius: 5px;">Open</span>';
+                                            span.parentElement.style.width = '125px';
+                                        }
+                                        break;
+                                    case 'Acknowledged':
+                                        if(columnIndex == 2  && span.parentElement?.style){
+                                            span.innerHTML = '<span style="border: 1px mediumblue solid;' +
+                                                'padding: 4px;' +
+                                                'border-radius: 5px;">Acknowledged</span>';
+                                            span.parentElement.style.width = '125px';
+                                        }
+                                        break;
+                                    case 'In_progress':
+                                        if(columnIndex == 2 && span.parentElement?.style){
+                                            span.innerHTML = '<span style="color: white;' +
+                                                'padding: 4px;' +
+                                                'background-color: green;' +
+                                                'border-radius: 5px;">In progress</span>';
+                                            span.parentElement.style.width = '125px';
+                                        }
+                                        break;
+                                    case 'Closed':
+                                        if(columnIndex == 2 && span.parentElement?.style){
+                                            span.innerHTML = '<span style="border: 1px grey solid;' +
+                                                'padding: 4px;' +
+                                                'border-radius: 5px;' +
+                                                'color: grey;">Closed</span>';
+                                            span.parentElement.style.width = '125px';
+                                        }
+                                        break;
+                                }
+                            });
```

Comment:
This code could be way more efficient haha. Using a Lit directive might help with applying styling dynamically; see [classMap](https://lit.dev/docs/templates/directives/#classmap) and [styleMap](https://lit.dev/docs/templates/directives/#stylemap) docs. Either way I would move this logic to a separate function for easier reusability.